README.md

```diff
@@ -1 +1,196 @@
 # Event-triggered Notifications Tutorial
+
+An event-triggered notification is an automated messages sent in response to an event.
+
+There are three approaches for implementing these types of notifications:
+
+- Leverage **Segments** for sending messages.
+- Use the OneSignal Notification API.
+- Use a platform SDK.
+
+This tutorial series will cover the three methods outlined above, but first, I need to cover the building blocks that make this possible: events, segments, and automated messages.
+
+## Example Use-case
+
+Suppose we want to send a new user an email, reminding them to complete the onboarding process for our app. The workflow may look something like the following:
+
+1. A potential user lands on our site.
+2. They sign up for our app.
+3. They begin the onboarding process for the app, but don't complete it.
+4. The app sends a customized email after 2 days of inactivity reminding them to complete onboarding.
+
+Each event results from a user action, and we can conditionally perform actions based on the event's context. You may be thinking that all this event stuff is a bit abstract, so we'll work through implementing the use case. Along the way, I'll cover some best practices to get the most out of event-triggered notifications in OneSignal.
+
+## What Are Events
+
+An event is an action or occurrence recognized by the software that usually originates from user interaction.
+
+Common events you may want to respond to include
+
+- Cart abandonment
+- User achievement
+- New user signup
```

Comment:
We should either use punctuation at the end of list items on this doc or don't -- the style guide doesn't have an opinion.  However we should be consistent.

Reply:
I like colons at the end of lines with illustrations following, and periods at the end of others.

Reply:
This should be consistent now.